pom.xml

```diff
@@ -92,14 +99,8 @@ Licensed under the MIT license. See License.txt in the project root. -->
         <includes>
           <include>Install.md</include>
           <include>ReadMe.md</include>
-        </includes>
```

Comment:
I just combined these 2 sections because they did the same thing for files in the same location.